clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/IngestionThrottler.java

```diff
@@ -104,22 +104,6 @@ public IngestionThrottler(
             "current_version_non_aa_wc_leader_records_count",
             false,
             EventThrottler.BLOCK_STRATEGY));
-    this.poolTypeRecordThrottlerMap.put(
```

Comment:
Can this be controlled by the config we added? Say, if the config is not enabled, we still use static throttler.